dtmi/com/socomec/socomecdevice-1.json

```diff
@@ -0,0 +1,1000 @@
+        "@id": "dtmi:com:socomec:socomecdevice:Load_2_10025;1",
+        "@type": [
+          "Telemetry",
+          "Current"
+        ],
+        "displayName": {
+          "en": "I3 Load 3"
+        },
+        "name": "Load_2_10025",
+        "schema": "float",
+        "unit": "ampere"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_0_10026;1",
+        "@type": [
+          "Telemetry",
+          "Current"
+        ],
+        "displayName": {
+          "en": "In Load 1"
+        },
+        "name": "Load_0_10026",
+        "schema": "float",
+        "unit": "ampere"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_1_10026;1",
+        "@type": [
+          "Telemetry",
+          "Current"
+        ],
+        "displayName": {
+          "en": "In Load 2"
+        },
+        "name": "Load_1_10026",
+        "schema": "float",
+        "unit": "ampere"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_2_10026;1",
+        "@type": [
+          "Telemetry",
+          "Current"
+        ],
+        "displayName": {
+          "en": "In Load 3"
+        },
+        "name": "Load_2_10026",
+        "schema": "float",
+        "unit": "ampere"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_0_80031;1",
+        "@type": [
+          "Telemetry",
+          "Power"
+        ],
+        "displayName": {
+          "en": "P tot Load 1"
+        },
+        "name": "Load_0_80031",
+        "schema": "float",
+        "unit": "watt"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_1_80031;1",
+        "@type": [
+          "Telemetry",
+          "Power"
+        ],
+        "displayName": {
+          "en": "P tot Load 2"
+        },
+        "name": "Load_1_80031",
+        "schema": "float",
+        "unit": "watt"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_2_80031;1",
+        "@type": [
+          "Telemetry",
+          "Power"
+        ],
+        "displayName": {
+          "en": "P tot Load 3"
+        },
+        "name": "Load_2_80031",
+        "schema": "float",
+        "unit": "watt"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_0_75011;1",
+        "@type": "Telemetry",
+        "displayName": {
+          "en": "PF tot Load 1"
+        },
+        "name": "Load_0_75011",
+        "schema": "float"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_1_75011;1",
+        "@type": "Telemetry",
+        "displayName": {
+          "en": "PF tot Load 2"
+        },
+        "name": "Load_1_75011",
+        "schema": "float"
+      },
+      {
+        "@id": "dtmi:com:socomec:socomecdevice:Load_2_75011;1",
+        "@type": "Telemetry",
+        "displayName": {
+          "en": "PF tot oad 3"
+        },
+        "name": "Load_2_75011",
+        "schema": "float"
+      },
+      {
+        "@id": "dtmi:com:socomec:S30000:Global_0_85880;1",
```

Comment:
All `@id` elements should use the same root, as described [here](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Resolution-Convention#unique-id-elements).

`dtmi:com:socomec:S30000:Global_0_85880;1` should be `dtmi:com:socomec:socomecdevice:S30000:Global_0_85880;1`

or you can also delete those IDs as they are not strictly required

Reply:
Thanks for the detail @rido-min !
I push new commit right now, should be OK (I have tested with both validation tool and IOT Central).